### PR TITLE
#13676. Add support for `sse` command

### PR DESCRIPTION
**Pull request by sergiohs84**

### CODE REVIEW

File: src/commands.cpp [hunk 1]
```diff
@@ -5308,6 +5308,28 @@ void CommandSendEvent::procresult()
     }
 }
 
+CommandSupportTicket::CommandSupportTicket(MegaClient *client, const char *message, int type)
+{
+    cmd("sse");
+    arg("t", type);
+    arg("m", message);
```
Comment (vtmateos) [inline src/commands.cpp]:
```suggestion
    arg("m", message, strlen(message));
```

Comment (sergiohs84) [reply src/commands.cpp]: Done, @vtmateos .

**Comment by aabellagm** [comment]
trigger compilation

**Comment by aabellagm** [comment]
trigger compilation

**Comment by aabellagm** [comment]
trigger compilation

**Comment by aabellagm** [comment]
trigger compilation

**Comment by aabellagm** [comment]
trigger compilation

**Pull request merged**